installers/installer.iss

```diff
@@ -2,8 +2,8 @@
 ; SEE THE DOCUMENTATION FOR DETAILS ON CREATING INNO SETUP SCRIPT FILES!
 
 #define MyAppName "SasView"
-#define MyAppVersion "5.0.5"
-#define MyAppPublisher "(c) 2009 - 2021, UTK, UMD, NIST, ORNL, ISIS, ESS, ILL, ANSTO, TU Delft and DLS"
+#define MyAppVersion "5.0.6b1"
```

Comment:
For the installer we should just use 5.0.6 (for the betas and the final release).  Betas should be overwritten by the final release in any case.  Once that is changed this should be ready to merge.